common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -1,276 +1,337 @@
-﻿has_base_giant_maa_trigger = {
+﻿can_unlock_innovation_giants_trigger = {
 	OR = {
 		#Races that are considered to have giants among their ranks
 		has_cultural_pillar = heritage_aqir
 		has_cultural_pillar = heritage_demonic
 		has_cultural_pillar = heritage_draconic
 		has_titanforged_heritage_trigger = yes
-		has_special_giant_maa_trigger = yes
 	}
 	NOT = { 
 		this = culture:tolvir
 		this = culture:mechagnome
 	}
 	is_giant_culture_trigger = no
+	can_unlock_culture_giant_maa_trigger = no
 }
-has_massive_aerial_maa_trigger = {
-	has_special_massive_aerial_maa_trigger = yes
+
+can_unlock_innovation_flying_beasts_trigger = {
+	OR = {
+		has_cultural_pillar = heritage_draconic
+		has_goblin_heritage_trigger = yes
+		has_cultural_pillar = heritage_pandaren
+		has_cultural_pillar = heritage_vrykulic
+		this = culture:dragonmaw
+		this = culture:dryad
+		this = culture:night_elf
+		this = culture:scourge
+	}
+	can_unlock_culture_massive_aerial_maa_trigger = no
+	# Needed for regional
+	has_unlocked_special_massive_aerial_maa_trigger = no
 }
 
-### Like triggers
-#Culture that prefers pikemen and dislike cavalry
-mountain_culture_trigger = {
+can_unlock_innovation_taming_the_skies_trigger = {
+	can_use_aerial_trigger = yes
+	can_unlock_culture_aerial_maa_trigger = no
+	# Needed for regional	
+	has_unlocked_special_aerial_maa_trigger = no	
+}
+
+can_unlock_innovation_archmage_trigger = {
 	OR = {
-		has_cultural_pillar = heritage_dwarven
-		has_cultural_pillar = heritage_gnomish
-		this = culture:alteraci
+		has_human_heritage_trigger = yes
+		this = culture:blood_elf
+		this = culture:high_elf
 	}
+	NOT = {
+		this = culture:pirate_culture
+		this = culture:northsea
+		this = culture:wastewander	
+	}
+}
+
+can_unlock_innovation_blademaster_trigger = {
+	this = culture:burning_blade
 }
-desert_culture_trigger = {
+
+can_unlock_innovation_buccaneer_trigger = {
 	OR = {
-		has_cultural_pillar = heritage_pygmy
-		has_cultural_pillar = heritage_sethralic
-		has_cultural_pillar = heritage_vulpine
-		this = culture:farraki
-		this = culture:tolvir
-		this = culture:wastewander
+		this = culture:pirate_culture
+		this = culture:northsea
+		this = culture:blackwater
 	}
 }
-#Culture that prefers aerial and dislike everything else
-winged_culture_trigger = {
+
+can_unlock_innovation_brute_trigger = {
+	has_cultural_pillar = heritage_hyenaic
+}
+
+can_unlock_innovation_candleholder_trigger = {
+	has_cultural_pillar = heritage_murine
+}
+
+can_unlock_innovation_centaur_tactics_trigger = {
+	this = culture:centaur
```

Comment:
Keep in mind many innovations become traditions.
And you can create a new culture so for example if you create a new centaur culture, they won't have centaur tactics.

Reply:
New cultures? No. You should use `is_culture_or_parent_culture_trigger` instead if you want to include cultures that divergent from the culture.